ndarray/src/main/java/org/tensorflow/ndarray/impl/sparse/window/ByteSparseWindow.java

```diff
@@ -0,0 +1,138 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.ndarray.impl.sparse.window;
+
+import org.tensorflow.ndarray.ByteNdArray;
+import org.tensorflow.ndarray.NdArray;
+import org.tensorflow.ndarray.NdArrays;
+import org.tensorflow.ndarray.buffer.ByteDataBuffer;
+import org.tensorflow.ndarray.buffer.DataBuffer;
+import org.tensorflow.ndarray.buffer.DataBuffers;
+import org.tensorflow.ndarray.impl.dimension.DimensionalSpace;
+import org.tensorflow.ndarray.impl.dimension.RelativeDimensionalSpace;
+import org.tensorflow.ndarray.impl.sparse.AbstractSparseNdArray;
+import org.tensorflow.ndarray.index.Index;
+
+import java.nio.ReadOnlyBufferException;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ByteSparseWindow extends SparseWindow<Byte, ByteNdArray> implements ByteNdArray {
```

Comment:
Can we rename this package and its classes to `slice` instead? (i.e. org.tensorflow.ndarray.impl.sparse.slice.ByteSparseSlice`)

Because `window` is already being used by `DataBufferWindow`, which is a different concept and act differently (allowing to "slide" a view inside a buffer)

Reply:
Ok